tests/generic_config_updater/test_syslog.py

```diff
@@ -2,66 +2,136 @@
 import pytest
 
 from tests.common.helpers.assertions import pytest_assert
-from tests.common.config_reload import config_reload
-from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success_and_reset_check, expect_res_success, expect_op_failure
+from tests.generic_config_updater.gu_utils import apply_patch, expect_res_success, expect_op_failure, expect_op_success
 from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
 
 pytestmark = [
     pytest.mark.topology('any'),
 ]
 
 logger = logging.getLogger(__name__)
 
-SYSLOG_TIMEOUT              = 10
-SYSLOG_INTERVAL             = 1
 # This is restricted by sonic-syslog.yang. Use '-1' to indicate no max is set
 SYSLOG_MAX_SERVER           = -1
 # The max server test only support SYSLOG_MAX_SERVER that is equal or lower than 254.
 SYSLOG_TEST_MAX_UPPER_LIMIT = 254
```

Comment:
>254 [](http://example.com/codeflow?start=30&length=3)

Where is 254 from? Is there any constrain or limitation by yang model or sonic implementation?

Reply:
removed cuz no constraint